src/GHCMod/Options/Commands.hs

```diff
@@ -60,6 +60,7 @@ data GhcModCommands =
   | CmdMapFile FilePath
   | CmdUnmapFile FilePath
   | CmdQuit
+  | CmdTest FilePath
```

Comment:
I would argue that this would be better-placed at line 59, since comment implies that `CmdTest` is interactive-only command (which it isn't)